Verifying-ERC20.md

```diff
@@ -247,7 +245,7 @@ When we invoke VeriMan we obtain the following counter example.
 
 which looks counter intuitive at the beginning. However, it is indeed true that the constructor changes the value of `_totalSupply` and 
 `VeriSol.Old` is implemented in a way that its value is undefined before creating the contract (which makes sense!).
-So, we slightly modify the contract to add a boolean variable that helps us avoid this particular case (the code is [here](https://github.com/garbervetsky/ERC-Verisol-Demo/blob/master/ERC20-Veriman.sol) and using this [config](https://github.com/garbervetsky/ERC-Verisol-Demo/blob/master/config.json) file for Veriman). 
+So, we slightly modify the contract to add a Boolean variable that helps us avoid this particular case (the code is [here](https://github.com/garbervetsky/ERC-Verisol-Demo/blob/master/ERC20-Veriman.sol) and using this [config](https://github.com/garbervetsky/ERC-Verisol-Demo/blob/master/config.json) file for VeriMan). 
 
 Now, we invoke VeriMan with this property `notConstructor -> (VeriSol.Old(_totalSupply) ==_totalSupply)` and obtain a new counter example. 
```

Comment:
I am surprised that the second counterexamples fails with [constructor, burn] but when you add || mintcalled, it passes. Although burn does not call mint, does it?